nimcoverage.nim

```diff
@@ -8,4 +8,5 @@ if getEnv("NIM_COVERAGE_DIR").len == 0:
     echo "NIM_COVERAGE_DIR environment variable not set"
     quit 1
 
+echo "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
```

Comment:
Is this required?

Reply:
Debug statement, must've missed it, sorry!